ansible/server-state-playbooks/ome-dundeeomero/templates/nightly-pg_dump-omero.sh.j2

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+# Clean the files from the folder-format dump directory 
+rm {{ omero_server_db_dumpdir_parent }}/{{ omero_server_db_dumpdir_name }}/* 
+# Create a fresh db dump
+sudo -u postgres pg_dump -j {{ (ansible_processor_count * ansible_processor_cores) |round|int }} -Fd -f {{ omero_server_db_dumpdir_parent }}/{{ omero_server_db_dumpdir_name }} {{ vault_omero_server_dbname }}
```

Comment:
Is that because you were divding by 2? https://github.com/openmicroscopy/infrastructure/blob/bb9eff03b7e18452424266a32e0d1691a94d0b23/ansible/server-state-playbooks/nightshade-web/playbook.yml#L71